### ISSUE TITLE
C support

### ISSUE THREAD

**Issue by terorie** [neodyme-labs/solana-security-txt#1]
It would be good to add support for the Solana C SDK

**Comment by tlambertz** [comment]
Agreed. Should be fairly simple to do. Out of interest, are you aware of any larger contracts except for Pyth that are written in C?
It seems of the currently ~2.5k deployed contracts, only 64 are written in C. (found via ELF contains `llvm-project.git`)

**Comment by terorie** [comment]
I am not aware of any other C contracts. Pyth is the only smart contract I've seriously worked with, so my view is very limited.

**Comment by mryalamanchi** [comment]
It should be possible using the __attribute__((section(".security.txt"))) to acheive that at code level it self!

**Comment by mryalamanchi** [comment]
> Agreed. Should be fairly simple to do. Out of interest, are you aware of any larger contracts except for Pyth that are written in C? It seems of the currently ~2.5k deployed contracts, only 64 are written in C. (found via ELF contains `llvm-project.git`)

Primary choice still remaind rust till date, except pyth ofcourse!

**Comment by MrG9090** [comment]
> Agreed. Should be fairly simple to do. Out of interest, are you aware of any larger contracts except for Pyth that are written in C? It seems of the currently ~2.5k deployed contracts, only 64 are written in C. (found via ELF contains `llvm-project.git`)

https://github.com/neodyme-labs/solana-security-txt/issues/1#issue-1167530593

**Comment by MrG9090** [comment]
B

**Comment by JanKortes212** [comment]
hi
